digestbot/core/db/models.py

```diff
@@ -9,7 +9,7 @@ class Category:
     channel_ids: List[str]
 
 
-@dataclass(frozen=True)
+@dataclass(frozen=False)
```

Comment:
Why?

Reply:
Because when I update permalinks I also work with Messages and need to update info.
Will change it later (see todo)